src/batch-client/index.ts

```diff
@@ -310,6 +313,10 @@ export class ZimbraBatchClient {
 			batch: false,
 			cache: false
 		});
+
+		if (options.requestAPI) {
```

Comment:
we should update readme with this option, so devs should know they can pass custom fetch function here

Reply:
Please have a look: https://github.com/Zimbra/zm-api-js-client/tree/PREAPPS-6008#provide-custom-fetch-api